general/macros.hpc

```diff
@@ -200,6 +200,16 @@ EOF\
 # element as argument. Do not use for compiler!
 %hpc_module_dep_base    %{hpc_module_dep_path}/%{hpc_module_pname}%{?hpc_ext}/
 
+# For extra python flavors
+%hpc_module_dep_extra() \
+%{lua: \
+local arg = rpm.expand("%1") \
+for extra in rpm.expand("%{" .. arg .. "_provides}"):gmatch("%S+") do \
```

Comment:
Like before, an %{<arg>_provides} only exists in a python multibuild context. Since this looks rather python specific, you may want to move this to the python macros and flag it as such in the name. If you see a value to have this macro beyond python, you may want to explain how it is used so a user understands it without having to dig deeply into the details of this macro.